src/storage-preview/azext_storage_preview/operations/account.py

```diff
@@ -316,3 +316,86 @@ def update_file_service_properties(cmd, instance, enable_delete_retention=None,
         params['protocol_settings'].smb.channel_encryption = channel_encryption
 
     return params
+
+
+def _generate_local_user(local_user, PermissionScope, SshPublicKey, permission_scope=None, ssh_authorized_key=None,
```

Comment:
Usually, we parse `nargs='+'` parameters in customized `Action`, for example:
https://github.com/Azure/azure-cli/blob/78f15dd339ef17afe0b357b77d9c0ba19b14ef2a/src/azure-cli/azure/cli/command_modules/storage/_validators.py#L1473-L1487